clients/admin-ui/src/features/common/table/v2/cells.tsx

```diff
@@ -69,7 +70,23 @@ export const RelativeTimestampCell = ({
     addSuffix: true,
   });
 
-  return <DefaultCell value={sentenceCase(timestamp)} />;
+  const formattedDate = formatDate(new Date(time));
+
+  return (
+    <Flex alignItems="center" height="100%">
+      <Tooltip label={formattedDate} hasArrow>
+        <Text
+          fontSize="xs"
+          lineHeight={4}
+          fontWeight="normal"
+          overflow="hidden"
+          textOverflow="ellipsis"
+        >
+          {timestamp}
```

Comment:
Can we leave this sentence-cased?  We want it to be consistent when it's this or "about 1 hour" or similar. 

![Screenshot 2024-08-12 at 11 00 46](https://github.com/user-attachments/assets/0403866d-8679-4a17-bca7-d7a059afec28)

Reply:
sure, yes, I didn't mean to change it. Fixing it now.